src/data_info/services/metadata.clj

```diff
@@ -107,8 +108,6 @@
         new-unit   (reserved-unit avu-map)
         attr       (:attr avu-map)
         value      (:value avu-map)]
-    (log/debug "Fixed Path:" fixed-path)
-    (log/debug "check" (true? (attr-value? cm fixed-path attr value)))
```

Comment:
I removed these logs since there's not much good reason for it to call out to `attr-value?` twice, and the logging is probably not helpful.